adal/src/main/java/com/microsoft/aad/adal/Logger.java

```diff
@@ -305,12 +305,11 @@ private static String getCodeName(ADALError code) {
         return "";
     }
 
-    private static String GetUTCdatetimeAsString() {
+    private static String GetUTCDateTimeAsString() {
```

Comment:
oh, I agree, the *dateTimeAsString should be *DateTimeAsString, no doubt on it.  What I'm saying is  method name shouldn't be start with uppercase. I'm surprised that AndroidStudio doesn't point out that the method name should start with lowercase. 

---
In reply to: [65662672](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65662672) [](ancestors = 65662672,65642931)